server/server.go

```diff
@@ -2365,3 +2375,15 @@ func parseURL(s string, useHTTPSByDefault bool) (*url.URL, error) {
 	}
 	return url.Parse(s)
 }
+
+func getProvenance(revision string) map[string]interface{} {
+	p := make(map[string]interface{})
+
+	p["revision"] = revision
```

Comment:
Can we make this conditional on revision being non-empty? This would be consistent with other API responses.